### ISSUE TITLE
[torrentday] Exception (torrentday): Request to Jackett server failed

### ISSUE THREAD

**Issue by KevinOlsthoorn** [Jackett/Jackett#6676]
Hello!

I've been using Jackett for quite some time now and it has been working really well!
Unfortunately, I noticed that since today the TorrentDay tracker wasn't working properly anymore when using the tracker from Sonarr.
When I checked the Jackett web UI and ran a test of the connection, the connection was OK according to Jackett.
I tried to edit the config by clicking the cog icon of the tracker but got the error "Request to Jackett server failed".
I decided to remove the tracker and re-add it, but I was presented with the same error when trying to add the TorrentDay tracker.

This issue doesn't occur with any of the other private and public trackers.

Does anyone have any idea what could be the problem here?

I'm using the docker image linuxserver/jackett.

Thanks!

(At the bottom of your Jackett Dashboard page) **Jackett version**:  0.12.863.0

-------------------------------

**Comment by garfield69** [comment]
have you tried any of the alternate site link addresses that the jackett dashboard config presents as options?
which one do you normally access with your browser running on the same machine as jackett?

if you cannot get any of the alternates to work then there may be more info in the log with the **enhanced** switch enabled so:

Please provide the **full enhanced log** file. 
1. scroll down to the bottom of the Jackett Dashboard and *tick* the *enhanced Logging* checkbox
2. scroll up a bit and *click* on the *apply server settings* button
3. repeat the test
4. find the *log.txt* file (linux `~/.config/Jackett/`, windows `%ProgramData%\Jackett`)
5. edit it with a plain text editor, redacting any personal details, usernames, passwords, passkeys, hashes etc.
6. save, then drag-drop it here for us to take a look at.

Thanks.

**Comment by Tuumke** [comment]
Can't edit or re-add the index.

`System.Net.Http.HttpRequestException: Resource temporarily unavailable
 ---> System.Net.Sockets.SocketException (11): Resource temporarily unavailable
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean allowHttp2, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.GetHttpConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.DecompressionHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.DiagnosticsHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at CloudflareSolverRe.ClearanceHandler.SendRequestAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at CloudflareSolverRe.ClearanceHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Jackett.Common.Utils.Clients.HttpWebClient2NetCore.Run(WebRequest webRequest) in /home/appveyor/projects/jackett/src/Jackett.Common/Utils/Clients/HttpWebClient2NetCore.cs:line 241
   at Jackett.Common.Utils.Clients.WebClient.GetString(WebRequest request) in /home/appveyor/projects/jackett/src/Jackett.Common/Utils/Clients/WebClient.cs:line 114
   at Jackett.Common.Indexers.BaseWebIndexer.RequestStringWithCookies(String url, String cookieOverride, String referer, Dictionary`2 headers) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 449
   at Jackett.Common.Indexers.TorrentDay.GetConfigurationForSetup() in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/TorrentDay.cs:line 123
   at Jackett.Server.Controllers.IndexerApiController.Config() in /home/appveyor/projects/jackett/src/Jackett.Server/Controllers/IndexerApiController.cs:line 79
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/appveyor/projects/jackett/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 73`

**Comment by KevinOlsthoorn** [comment]
Hmm, alright. I think this problem is not specific to TorrentDay.
I tried applying the extended logging but when clicking "Apply server settings" I got the same error as when adding or editing the TorrentDay indexer...

The logs say the following:

`System.Exception: Error parsing JS challenge html
   at CloudflareSolverRe.Types.Javascript.JsChallenge.Parse(String html, Uri siteUrl)
   at CloudflareSolverRe.Solvers.JsChallengeSolver.SolveChallenge(String html)
   at CloudflareSolverRe.Solvers.JsChallengeSolver.Solve()
   at CloudflareSolverRe.CloudflareSolver.SolveJavascriptChallenge(Nullable`1 jsDetectResult)
   at CloudflareSolverRe.CloudflareSolver.SolveWithJavascript(Int32 tries)
   at CloudflareSolverRe.CloudflareSolver.Solve()
   at CloudflareSolverRe.CloudflareSolver.Solve(HttpClient httpClient, CloudflareHandler cloudflareHandler, Uri siteUrl, CancellationToken cancellationToken)
   at CloudflareSolverRe.CloudflareSolver.<>c__DisplayClass30_0.<<Solve>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at CloudflareSolverRe.Utilities.SemaphoreLocker.LockAsync[T](Func`1 worker)
   at CloudflareSolverRe.CloudflareSolver.Solve(HttpClient httpClient, HttpClientHandler httpClientHandler, Uri siteUrl, CancellationToken cancellationToken, Boolean randomUserAgent)
   at CloudflareSolverRe.ClearanceHandler.GetClearance(HttpRequestMessage request, CancellationToken cancellationToken)
   at CloudflareSolverRe.ClearanceHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Jackett.Common.Utils.Clients.HttpWebClient2NetCore.Run(WebRequest webRequest) in /home/appveyor/projects/jackett/src/Jackett.Common/Utils/Clients/HttpWebClient2NetCore.cs:line 241
   at Jackett.Common.Utils.Clients.WebClient.GetString(WebRequest request) in /home/appveyor/projects/jackett/src/Jackett.Common/Utils/Clients/WebClient.cs:line 165
   at Jackett.Common.Indexers.BaseWebIndexer.RequestStringWithCookies(String url, String cookieOverride, String referer, Dictionary`2 headers) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 449
   at Jackett.Common.Indexers.TorrentDay.GetConfigurationForSetup() in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/TorrentDay.cs:line 123
   at Jackett.Server.Controllers.IndexerApiController.Config() in /home/appveyor/projects/jackett/src/Jackett.Server/Controllers/IndexerApiController.cs:line 79
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Builder.Extensions.UsePathBaseMiddleware.Invoke(HttpContext context)
   at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/appveyor/projects/jackett/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 73`

Any idea what could be causing this?

**Comment by garfield69** [comment]
@KevinOlsthoorn wrote:
> Error parsing JS challenge html at CloudflareSolverRe

Ah. there is your problem. your jackett 0.12.863 is using CloudflareSolverRe 1.0.5
you need to upgrade jackett.  version 0.12.1042 was upgraded 26 days ago to use CloudflareSolverRe 1.0.6 which can handle the new cloudflare challenges.
The latest jackett version is currently 0.12.1354

**Comment by garfield69** [comment]
@Tuumke  you are hijacking this ticket. your problem is different. you should really open your own.
I have seen resource temporarily unavailable tickets, all appear to happen to users of Docker Jackett.
almost always its a DNS issue, and usually following https://development.robinwinslow.uk/2016/06/23/fix-docker-networking-dns/ helps to resolve.
OR
they get the next docket update and the problem magically resolves itself.
Basically, its nothing to do with jackett.

**Comment by Tuumke** [comment]
> @Tuumke you are hijacking this ticket. your problem is different. you should really open your own.
> I have seen resource temporarily unavailable tickets, all appear to happen to users of Docker Jackett.
> almost always its a DNS issue, and usually following https://development.robinwinslow.uk/2016/06/23/fix-docker-networking-dns/ helps to resolve.
> OR
> they get the next docket update and the problem magically resolves itself.
> Basically, its nothing to do with jackett.

Why should i create my own issue when i too have issues with torrentday and it says "Request to Jackett server failed" when trying to edit or re-add the Indexer in jackett... I just posted my piece of my log..

**Comment by KevinOlsthoorn** [comment]
Great, I've updated to Jackett 9.12.1354.0 and am able to open the configure window for the TorrentDay indexer.
Unfortunately, whenever I press the "Okay" button after filling in my cookies, nothing seems to happen.
The text in the button is replaced by a spinner, but it keeps spinning and nothing is happening.
I've tried multiple URLs as listed at the top of the configure window, but all seem to do the same thing.

I also noticed that saving Jackett's configuration still results in the same "Request to Jackett server failed" error.
Because of this I am unable to enable advanced logging like you requested.
The regular logs don't contain any errors anymore.

Any ideas?

Thanks for your help thus far.

**Comment by garfield69** [comment]
when you filled in the cookie did you use this method:
- Login to torrentday with your browser on the same machine that is running jackett
- Open the _DevTools_ panel by pressing **F12**
- Select the **Network** tab
- Click on the **Doc** button
- Refresh the page by pressing **F5**
- Select the **Headers** tab
- Find 'cookie:' in the **Request Headers** section
- Copy & paste/replace the whole cookie string (everything after `cookie: ` to the cookie box in the Jackett torrentday config.

**Comment by garfield69** [comment]
and when you login to torrentday with your browser, make sure to use the same site link address as you have chosen to use in the jackett torrentday config.

**Comment by KevinOlsthoorn** [comment]
> when you filled in the cookie did you use this method:
> 
> * Login to torrentday with your browser on the same machine that is running jackett
> * Open the _DevTools_ panel by pressing **F12**
> * Select the **Network** tab
> * Click on the **Doc** button
> * Refresh the page by pressing **F5**
> * Select the **Headers** tab
> * Find 'cookie:' in the **Request Headers** section
> * Copy & paste/replace the whole cookie string (everything after `cookie: ` to the cookie box in the Jackett torrentday config.

Yes, this is what I've done. Except for the exact machine I've done this on. I'm unable to do this on the machine that is running Jackett, since that is a headless machine and therefore doesn't have Chrome installed.
This didn't seem to be problem before, when it still worked.

I've also used the same domain for obtaining the cookies. Different domains didn't make a difference unfortunately.

**Comment by garfield69** [comment]
Ah, another problem involving a headless server. seem to be running into this more and more recently. #6661 
Sorry I can't help you further. Maybe a volunteer with more experience in the area can help.

**Comment by Tuumke** [comment]
I managed to get my problems fixed and was able to add TD again. Didn't use the example cookie format that is auto filled but used the one from the "how-to" page. Basically you set the url, copy paste it. Press F12 on a new tab and paste the url. If logged in press logout and log back in. Then use the uid, pass and __..uid so it looks like example page. If it ain't working, I could help with any desk (remote)

**Comment by KevinOlsthoorn** [comment]
Thanks for your reply, but that doesn't seem to work for me either. It seems like it doesn't matter what I fill in in the cookies field of the TD config page in Jackett. The Okay button always changes to a spinner and keeps spinning indefinitely. This happens when I fill in the correct cookies but also when I don't.

**Comment by Tuumke** [comment]
Okai. Then i dont think i can help. FWI: 
My cookie in Jackett looks like:
`uid=000000; pass=longstring; __cfduid=evenlongerstring; cf_clearance=superlongstring`

**Comment by Tuumke** [comment]
Another thing you could try is:

1. stop docker container

**Comment by KevinOlsthoorn** [comment]
Thanks for the suggestion. Following these steps eventually resulted in me realizing this problem has probably to do with my docker-vpn setup causing some specific traffic of Jackett to fail (see some of comments above). I was using a reverse proxy to access Jackett too. When disabling the reverse proxy and vpn client I was able to properly setup the TorrentDay indexer. After re-enabling the reverse proxy and vpn client, the indexer still worked (allthough I'm not able to update it, but that's okay).

Since this issue (I think) has nothing to do with Jackett itself, but more with my reverse proxy and vpn setup, I'll close this issue.

Thanks for the help.

**Comment by bombz88** [comment]
Hello, 
I have been looking all over the net and cannot for the life of me get jacket to see TD. I think the jackett URL is incorrect as I cannot add it at all. There is no way to edit this.
Any step by steps on how to add TD to jackett?

**Comment by bombz88** [comment]
> Another thing you could try is:
> 
> 1. stop docker container
> 2. move jackett folder to jackett_old
> 3. docker pull linuxserver/jackett
> 4. docker rm jackett
> 5. docker create command you used for jackett
> 6. docker start jackett
> 7. try to add it then

Hello, I cannot edit jacket from the web UI, any other ideas?

**Comment by garfield69** [comment]
@bombz88  try the following back-door method
1. user a browser on the same machine that is running Jackett and going through the same VPN/proxy as jackett if you are using one.
2. check to see which of these domains works
```
https://tday.love/
https://torrentday.cool/
https://tdonline.org/
https://secure.torrentday.com/
https://torrentday.eu/
https://classic.torrentday.com/
https://www.torrentday.com/
https://td-update.com/
https://www.torrentday.me/
https://www.torrentday.ru/
https://www.td.af/
https://torrentday.it/
https://td.findnemo.net/
https://td.getcrazy.me/
https://td.venom.global/
https://td.workisboring.net/
```
3. shut down the Jackett service 
4. using a plain text editor paste the following and save to your jackett **Indexers** folder as `TorrentDay.json`
```json
[
  {
    "id": "sitelink",
    "type": "inputstring",
    "name": "Site Link",
    "value": "your-alternate-domain-goes-here"
  },
]
```
replace `your-alternate-domain-goes-here` with the domain that you found works for you
on most linux systems the **Indexers** folder is in `~/.config/Jackett/Indexers`, on windows `%ProgramData%\Jackett\Indexers `
5. replace your-alternate-domain-goes-here
6. start the service
7. try to add/edit the TorrentDay config from the Jackett dashboard and see if you get lucky.

**Comment by bombz88** [comment]
> @bombz88 try the following back-door method
> 
> 1. user a browser on the same machine that is running Jackett and going through the same VPN/proxy as jackett if you are using one.
> 2. check to see which of these domains works
> 
> ```
> https://tday.love/
> https://torrentday.cool/
> https://tdonline.org/
> https://secure.torrentday.com/
> https://torrentday.eu/
> https://classic.torrentday.com/
> https://www.torrentday.com/
> https://td-update.com/
> https://www.torrentday.me/
> https://www.torrentday.ru/
> https://www.td.af/
> https://torrentday.it/
> https://td.findnemo.net/
> https://td.getcrazy.me/
> https://td.venom.global/
> https://td.workisboring.net/
> ```
> 
> 1. shut down the Jackett service
> 2. using a plain text editor paste the following and save to your jackett **Indexers** folder as `TorrentDay.json`
> 
> ```json
> [
>   {
>     "id": "sitelink",
>     "type": "inputstring",
>     "name": "Site Link",
>     "value": "your-alternate-domain-goes-here"
>   },
> ]
> ```
> 
> replace `your-alternate-domain-goes-here` with the domain that you found works for you
> on most linux systems the **Indexers** folder is in `~/.config/Jackett/Indexers`, on windows `%ProgramData%\Jackett\Indexers `
> 5. start the service
> 6. try to add/edit the TorrentDay config from the Jackett dashboard and see if you get lucky.

I will try this. I was not sure where to start to edit the URL for jackett. Let me see if I can follow this guide and have some success with adding it.
You understand the concern I am talking about?
jackett points to - https://tday.love/
there is no add button 
when you select the wrench jackett tosses the error in the UI
"request to jackett server failed"

**Comment by garfield69** [comment]
when you click on the wrench to add an indexer, the first thing Jackett does is go to the default SiteLink and fetch the login page, so that it can pre-load any config boxes with values you may need. 
But if the cloudflare anti-DDoS interferes with this fetch, then the indexer config panel cannot be displayed and you cannot fill in the login username/password and whatnot.
The .json file in the Indexers folder is where Jackett stores anything that gets saved from the indexer config panel **after** you successfully login.
What you are doing by preparing a .json ahead of time, is telling Jackett what the default sitelink should be for this indexer, and hopefully, if you have found one that does not give you a challenge, Jackett will be able to present a config panel for you so you can load your login credentials.

clear as mud?

As for the Jackett config, this is automatically created when you first start Jackett, and all its values come from the Jackett dashboard options on the home page.
you do not need to manually edit this file for any reason other than to recover from a disaster.
When ever you change an option on the Jackett dashboard and click **apply server settings** the file is updated.

**Comment by bombz88** [comment]
I followed your guide, and torrentday was automatically in the indexers when I went to the Jackett UI. However, I am getting a cookie error now. I went to the same website, and followed the method to copy the cookie over bit am receiving this error
**An error occurred while updating this indexer
Your cookie did not work: Parse error**

I made sure to copy the cooking a few times to make sure I didn't miss anything but received the same result

**Comment by garfield69** [comment]
For the cookie, as long as you are following https://github.com/Jackett/Jackett/wiki/Troubleshooting#your-cookie-did-not-work you should have the correct string.

what platform are you running Jackett on?
what version of jackett are you running?
is the Jackett build a .net core or a mono depended one?

some users on linux, and especially if they are using proxies, cannot get cookies to work.
Also linux mono users have trouble.

and for those two combinations there is no solution.

**Comment by bombz88** [comment]
> For the cookie, as long as you are following https://github.com/Jackett/Jackett/wiki/Troubleshooting#your-cookie-did-not-work you should have the correct string.
> 
> what platform are you running Jackett on?
> what version of jackett are you running?
> is the Jackett build a .net core or a mono depended one?
> 
> some users on linux, and especially if they are using proxies, cannot get cookies to work.
> Also linux mono users have trouble.
> 
> and for those two combinations there is no solution.

 For the cookie, as long as you are following https://github.com/Jackett/Jackett/wiki/Troubleshooting#your-cookie-did-not-work you should have the correct string.
**https://github.com/Jackett/Jackett/wiki/Finding-cookies**
 
what platform are you running Jackett on?
**Windows 10**
what version of jackett are you running?
**Jackett Version 0.16.7.0**
is the Jackett build a .net core or a mono depended on one
**unsure how to answer this :-(** 
some users on linux, and especially if they are using proxies, cannot get cookies to work.
 Also linux mono users have trouble.
**not using unix**

 and for those two combinations, there is no solution.
**gotcha**

**Comment by bombz88** [comment]
> @bombz88 try the following back-door method
> 
> 1. user a browser on the same machine that is running Jackett and going through the same VPN/proxy as jackett if you are using one.
> 2. check to see which of these domains works
> 
> ```
> https://tday.love/
> https://torrentday.cool/
> https://tdonline.org/
> https://secure.torrentday.com/
> https://torrentday.eu/
> https://classic.torrentday.com/
> https://www.torrentday.com/
> https://td-update.com/
> https://www.torrentday.me/
> https://www.torrentday.ru/
> https://www.td.af/
> https://torrentday.it/
> https://td.findnemo.net/
> https://td.getcrazy.me/
> https://td.venom.global/
> https://td.workisboring.net/
> ```
> 
> 1. shut down the Jackett service
> 2. using a plain text editor paste the following and save to your jackett **Indexers** folder as `TorrentDay.json`
> 
> ```json
> [
>   {
>     "id": "sitelink",
>     "type": "inputstring",
>     "name": "Site Link",
>     "value": "your-alternate-domain-goes-here"
>   },
> ]
> ```
> 
> replace `your-alternate-domain-goes-here` with the domain that you found works for you
> on most linux systems the **Indexers** folder is in `~/.config/Jackett/Indexers`, on windows `%ProgramData%\Jackett\Indexers `
> 5. start the service
> 6. try to add/edit the TorrentDay config from the Jackett dashboard and see if you get lucky.

You are seriously the BEST man! 
I pulled some data from sonarr, and see the trackers lighting up for TD within my client. 
This was the perfect guide!
Skills that pay the billz brother!!
Thank you so much! I will keep this thread available if anything else comes up!
